src/lib/y2users/users_simple/writer.rb

```diff
@@ -44,8 +44,7 @@ def write
         root = config.users.find(&:root?)
         return unless root
 
-        # TODO: save plain password instead of encrypted one.
-        Yast::UsersSimple.SetRootPassword(root.password.value) unless root.password.nil?
+        Yast::UsersSimple.SetRootPassword(root.password.plain_value) unless root.password.nil?
```

Comment:
Please, see the next TODO at line 69. It should be adapted too.

Reply:
good catch